src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -137,6 +142,11 @@ public boolean doneScoring() {
     return true; // Placeholder until logic is made
   }
 
+  /** Returns the cosine of the arm angle in degrees off of the horizontal. */
+  public double getCosineArmAngle() {
+    return Math.cos(armEncoder.getPosition() - 90);
```

Comment:
This 90 and the 180 in the other subsystem are "magic numbers." It's not obvious why the number is what it is. Should be a named variable that makes it clear what it's doing